examples/templates/README.md

```diff
@@ -1,21 +1,45 @@
 ## Templates Guidline
 
-These templates are a base for the new code examples.
+This folder contains templates for new code examples. Every new code example should be made using these templates. 
```

Comment:
Every new code example should be made using these templates. 

WDYT "Each new code example should be created using the template from this folder".